dataformats/multigrid/src/RunSpecParse.cpp

```diff
@@ -25,29 +25,33 @@ std::vector<RunSpec *> &RunSpecParse::getruns(std::string runspec,
     outputdir += '/';
   }
 
-  if (!reader.parse(str, root, 0)) {
-    printf("error: file \"%s\" is not valid json\n", jsonfile.c_str());
+  nlohmann::json root;
+  try {
+    root = nlohmann::json::parse(jsonstring);
```

Comment:
You should really start using the LLVM coding standard.

Reply:
so Root instead of root?

Reply:
Yes.